Tasks/NuGetInstaller/nugetinstaller.ts

```diff
@@ -25,10 +25,23 @@ if(!nuGetPathToUse) {
 }
 tl.checkPath(nuGetPathToUse, 'nuget');
 
+//check if mono is available,
+var monoPath = tl.which('mono');
+if(!monoPath) {
+    tl.error('Failed to find mono. Mono is required to run NuGet.exe on OSX and Linux.');
+    tl.exit(1);
+}
```

Comment:
Won't this break on Windows if mono is in the path?  It will result in mono being used to run NuGet.exe on Windows, which I expect will fail.